### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JamesH65**
Added links to own brand power supplies, and some minor changes to text to include Pi4

### CODE REVIEW

File: faqs/README.md [hunk 1]
```diff
@@ -393,7 +393,7 @@ Power interruptions can cause problems on all sorts of electronic devices, and t
...
+The device is powered by 5V micro USB. Exactly how much current (mA) the Raspberry Pi requires is dependent on which model you are using, and what you hook up to it. We recommend our own power supplies, and sell a [2.5A (2500mA)](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) supply for models up to the Pi 3, and a [3.0A (3000mA) supply](https://www.raspberrypi.org/products/type-c-power-supply/) for the Pi 4. These will provide you with enough power to run your Raspberry Pi for most applications, including use of the 4 USB ports. Very high-demand USB devices may however require the use of a powered hub. The table below outlines the specific power requirements of each model.
```
[the remark this answers is not in the document]
Comment (lurch) [reply faqs/README.md]: Also, the next sentence makes it sound like the _only_ difference between the Pi3 and Pi4 power supplies is that the Pi3 PSU is up to 2.5A and the Pi4 PSU is up to 3.0A; so this sentence could probably do with a bit more clarification too.

File: faqs/README.md [hunk 2]
```diff
@@ -409,7 +409,11 @@ The device is powered by 5V micro USB. Exactly how much current (mA) the Raspber
...
+
+The Raspberry Pi  B+ and 2 Model B can supply 600mA/1.2A to downstream USB peripherals, switchable by a firmware setting. This allows the vast majority of USB devices to be connected directly to these models, assuming the upstream power supply has sufficient available current. The Raspberry Pi 3 and 4 provide 1.2A by default. 
+
+Very high-current devices or devices which can draw a surge current such as certain modems and USB hard disks will still require an external powered USB hub. The power requirements of the Raspberry Pi increase as you make use of the various interfaces on the Raspberry Pi. The GPIO pins can draw 50mA safely (note that that means 50mA distributed across all the pins: an individual GPIO pin can only safely draw 16mA), the HDMI port uses 50mA, the Camera Module requires 250mA, and keyboards and mice can take as little as 100mA or as much as 1000mA! Check the power rating of the devices you plan to connect to the Raspberry Pi and purchase a power supply accordingly. If you're not sure, we would advise you to buy a powered hub.
```
[the remark this answers is not in the document]
Comment (lurch) [reply faqs/README.md]: Probably doesn't hurt to change "... buy a powered hub." to "... buy a powered USB hub." ?

File: faqs/README.md [hunk 3]
```diff
@@ -409,7 +409,11 @@ The device is powered by 5V micro USB. Exactly how much current (mA) the Raspber
...
+The specific current requirements of each model are dependent on the use case: the PSU recommendations are based on **typical maximum** current consumption, the typical current consumption is for each board in a *desktop computer* configuration. The Raspberry Pi Model A, A+, and B can supply a maximum of 500mA to downstream USB peripherals. If you wish to connect a high-power USB device, it is recommended that you connect a powered USB hub to the Raspberry Pi and connect your peripherals to the USB hub.
+
+The Raspberry Pi B+ and Raspberry Pi 2B can supply 600mA/1.2A to downstream USB peripherals, switchable by a firmware setting. This allows the vast majority of USB devices to be connected directly to these models, assuming the upstream power supply has sufficient available current. The Raspberry Pi 3 and 4 provide 1.2A by default.
```
Comment (lurch) [inline faqs/README.md:414]: "switchable by a firmware setting"

I just cross-checked with https://github.com/raspberrypi/documentation/blob/master/configuration/config-txt/misc.md (I was going to suggest adding a link) and it seems that 1.2A is now the default on all models, so I guess this paragraph needs to be reworded?

**Comment by JamesH65** [comment]
@lurch OK to merge?

**Comment by lurch** [comment]
Should the "600mA/1.2A (switchable)" entries in the table be changed to just "1.2A" ?

**Pull request merged**